test/libsolidity/smtCheckerTests/loops/do_while_bmc_iterations_nested_continue.sol

```diff
@@ -0,0 +1,24 @@
+contract C
+{
+	function f(uint z) public pure {
+		uint x = 0;
+		require(z == 0);
+		do {
+			uint y = 0;
+			do {
+				++y;
+				if (y > 0)
+					continue;
+				++z;
+			} while (y < 2);
+			++x;
+		} while (x < 2);
+		assert(z == 0);
```

Comment:
Shouldn't this fail?

Reply:
@leonardoalt No it's all good.
`z` is never incremented, `continue` in the inner loop prevents that.